README.md

```diff
@@ -192,8 +192,9 @@ class MyVCFAnnotator(VCFAnnotator):
         :param vcf_template: vcf reader object
         :return:
         """
-        vcf_template.infos['gnomad_exomes_AN'] = _Info('gnomad_exomes_AN', '.', 'Integer',
-                                                             'GnomAD exomes allele number value', None, None)
+        vcf_template.infos['gnomad_exomes_AN'] = _Info('gnomad_exomes_AN', 1, 'Integer',
```

Comment:
Should the second parameter change from '.' to 1? If yes ignore my comment. If no, there are other places in the code that this has changed